garloff@suse.de

Feature.

Don't use REPORT_LUNS for scanning of SCSI devices connected to
the zfcp SCSI adapter (S/390).
Patch depends on scsi-scan-host-noreportlun.

--- linux-2.6.5/drivers/s390/scsi/zfcp_scsi.c.orig	2004-04-14 20:20:22.000000000 +0200
+++ linux-2.6.5/drivers/s390/scsi/zfcp_scsi.c	2004-04-16 19:59:03.960246802 +0200
@@ -774,6 +774,7 @@
 	adapter->scsi_host->max_channel = 0;
 	adapter->scsi_host->unique_id = unique_id++;	/* FIXME */
 	adapter->scsi_host->max_cmd_len = ZFCP_MAX_SCSI_CMND_LENGTH;
+	adapter->scsi_host->no_reportlun = 1;
 	/*
 	 * Reverse mapping of the host number to avoid race condition
 	 */
